src/main/java/nextstep/subway/error/ExceptionHandlerAdvice.java

```diff
@@ -20,9 +20,15 @@
         return ResponseEntity.badRequest().build();
     }
 
-    @ExceptionHandler(SubwayException.class)
-    public ResponseEntity<ErrorResponse> handleCommonException(SubwayException e) {
+    @ExceptionHandler(SubwayInternalException.class)
+    public ResponseEntity<ErrorResponse> handleSubwayInternalException(SubwayInternalException e) {
         ErrorResponse errorResponse = new ErrorResponse(HttpStatus.INTERNAL_SERVER_ERROR.value(), e.getMessage());
         return new ResponseEntity(errorResponse, errorResponse.toHttpStatus());
     }
+
+    @ExceptionHandler(SubwayBadRequestException.class)
+    public ResponseEntity<ErrorResponse> handleSubwayBadRequestException(SubwayInternalException e) {
```

Comment:
```suggestion
    public ResponseEntity<ErrorResponse> handleSubwayBadRequestException(SubwayBadRequestException e) {
```

잘못된 클래스로 처리를 하고 있는 것 같아요..!